patches/node/fast_path_for_adding_props_with_existing_transition.patch

```diff
@@ -1,80 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Shelley Vohr <shelley.vohr@gmail.com>
-Date: Mon, 11 Mar 2024 16:11:02 +0100
-Subject: Fast path for adding props with existing transition
-
-Adapt Node.js tests for https://chromium-review.googlesource.com/c/v8/v8/+/5342763.
-
-This patch can be removed when Node.js handles these changes.
```

Comment:
Not a review issue, just asking conversationally -- if this patch was going to need a Node.js change before it could be removed, how is it getting removed in a Chromium bump? Did Chromium do something that made this patch moot?

Reply:
@ckerr they reverted the original CL (linked in commit)

Reply:
gotcha :smile_cat: